### PR TITLE
plt-383 - Slack upload UI improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look.
Just making the import button disabled if a file is not uploaded and then enabling it once a file is uploaded.

### CODE REVIEW

File: web/react/components/setting_upload.jsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ export default class SettingsUpload extends React.Component {
         }
         $(e.target).closest('li').find('.file-status').addClass('hide');
         $(e.target).closest('li').find('.file-name').removeClass('hide').html(filename);
+        $(React.findDOMNode(this.refs.importButton)).removeClass('disabled');
```
Comment (crspeller) [inline web/react/components/setting_upload.jsx]: We should be using react changes in state to render rather than jquery magic.

**Comment by crspeller** [comment]
+1 Looks good.

**Comment by jwilander** [comment]
+1

**Comment by jwilander** [comment]
Waiting on Travis to merge

**Pull request merged**